README.md

```diff
@@ -31,14 +31,28 @@ allprojects {
 
 Add the project as a dependency:
 
-```java
+```
 dependencies {
 	compile 'com.github.stirante:lol-client-java-api:1.2.3'
 }
 ```
 
 ### Maven
 
+
+#### 1. Nexus Repo
+The public nexus repository can be access directly from maven you will just require the dependency to access any version.
+```xml
+	<!-- https://github.com/stirante/lol-client-java-api -->
+        <dependency>
+            <groupId>com.stirante</groupId>
+            <artifactId>lol-client-java-api</artifactId>
+            <version>1.2.11-SNAPSHOT</version>
+            <scope>provided</scope>
```

Comment:
I don't think scope is needed here. It would indicate, that the target application, that uses this library will already have somewhere in classpath this library and in most cases, that's not the case.

Reply:
my bad, copy/paste typo.